wasm/jsapi/functions/entry-different-function-realm.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Entry settings object for host functions when the function realm is different from the test realm</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/wasm/jsapi/wasm-module-builder.js"></script>
+<script src="/wasm/jsapi/function/helper.js"></script>
```

Comment:
I might be missing something obvious, but this file doesn't seem to exist?

Reply:
It did locally! Fixed.